### ISSUE TITLE
[🐞] 0.102.0 fails to upload to the AppStore

### ISSUE THREAD

**Issue by joshuapoq** [paypal/paypalcheckout-ios#108]
### 🐞 Describe the Bug
Version 0.102.0 fails to upload to the AppStore with Xcode 13 and 14.
```
Asset validation failed
Invalid directory. The bundle Payload/{App}.app/Frameworks/
PayPalCheckout.framework/PartnerAuthSDK.framework is not contained in a correctly named
directory. It should be under Frameworks. (ID: 18a2666-441b-4644-bc58-d3f9f36d3481)

Asset validation failed
Unexpected CFBundleExecutable Key. The bundle at 'Payload/{App}.app/Frameworks/
PayPalCheckout.framework/PartnerAuthSDK.framework/
PavPalAuthKitLocalizationBundle.bundle' does not contain a bundle executable. If this bundle
intentionally does not contain an executable, consider removing the CFBundleExecutable key
from its Info.plist and using a CFBundlePackageType of BNDL. If this bundle is part of a third-
party framework, consider contacting the developer of the framework for an update to address
this issue. (ID: 31b56b98-fc6f-4da2-9ef5-7f16e9059f63)
```

### 🔬 Minimal Reproduction
We have a Package that depends on PayPalCheckout.
Our apps depend on that Package.

### 🌍 Environment

- iOS version: iOS 14 +
- SDK version (`Checkout.version`): 0.102.0

**Comment by VSharkO** [comment]
Same here :/

**Comment by jonathajones** [comment]
Thanks for the report, we're addressing this.

**Comment by jonathajones** [comment]
Hi @joshuapoq + @VSharkO  - 
This should be resolved with the latest version (0.103.0). I tested validating the app as part of the submission process, and was able to reproduce the issue and then see it resolved after the fix. Let us know what your experience is.

Thank you!

**Comment by VSharkO** [comment]
For me issue is still present in 0.103.0 version

**Comment by glja021** [comment]
Facing the same issue on the latest release [0.103.0](https://github.com/paypal/paypalcheckout-ios/releases/tag/0.103.0) 
[0.101.0](https://github.com/paypal/paypalcheckout-ios/releases/tag/0.101.0) is not affected by this

**Comment by jonathajones** [comment]
> Facing the same issue on the latest release [0.103.0](https://github.com/paypal/paypalcheckout-ios/releases/tag/0.103.0) [0.101.0](https://github.com/paypal/paypalcheckout-ios/releases/tag/0.101.0) is not affected by this

This is right - we've identified the issue and are working on a resolution. In the meantime, please use version `0.101.0`.

Very sorry about this - we're working on making a more permanent solution available soon.

**Comment by vradopp** [comment]
@VSharkO @glja021 @joshuapoq These issues should now be fixed on version `0.104.0` Let us know if it is working for you.

Thanks!
